### PR TITLE
document active imperative mood for changelog messages

### PR DESCRIPTION
**Pull request by tgummerer**
As we discussed in ideation yesterday, this is the way we want to go, and try to be consistent with it.  I've also opened https://github.com/pulumi/go-change/pull/25 to update the go-change tool to suggest that mood.

### CODE REVIEW

**Review by Frassle — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -134,6 +134,19 @@ $ make changelog
 ````
 …and follow the prompts on screen.
 
+### Changelog grammatical mood
```
Comment (justinvp) [inline CONTRIBUTING.md:137]:
```suggestion
### Changelog messages
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -134,6 +134,19 @@ $ make changelog
...
+
+Changelog notes are written in the active imperative mood.  They should not end with a period.
```
Comment (justinvp) [inline CONTRIBUTING.md:139]:
```suggestion
Changelog notes are written in the active imperative form.  They should not end with a period.
```

Comment (justinvp) [inline CONTRIBUTING.md:139]: Also, would it be worth including the rule of thumb that Fraser had mentioned? I personally find it helpful. Something like:

> The simple rule is to pretend the message starts with "This change will ..."

**Review by justinvp — APPROVED** [review]
Some nitpicks

File: CONTRIBUTING.md (hunk 2 above)
Comment (tgummerer) [reply CONTRIBUTING.md]: yeah good idea, I'll add that

**Pull request merged**